File: packages/core/test/fixtures/tree-sitter/go/launchdarkly/positive/single-line.go
package main

import (
	ld "github.com/launchdarkly/go-server-sdk/v7"
)

func main() {
	client, _ := ld.MakeClient("sdk-key", 5)
	defer client.Close()

	enabled, _ := client.BoolVariation("GO_CHECKOUT_V2", ldcontext.New("user"), false)
	if enabled {
		println("v2")
	}
}

File: packages/core/test/fixtures/tree-sitter/java/launchdarkly/positive/single-line.java
import com.launchdarkly.sdk.server.LDClient;
import com.launchdarkly.sdk.LDContext;

public class Example {
    public void run(LDClient client, LDContext ctx) {
        boolean enabled = client.boolVariation("JAVA_CHECKOUT_V2", ctx, false);
        if (enabled) {
            System.out.println("v2");
        }
    }
}
